app/view/grid/Grid.js

```diff
@@ -236,5 +237,12 @@ Ext.define('CpsiMapview.view.grid.Grid', {
                 boxLabel: 'Page Records?',
                 handler: 'togglePaging'
             }]
-    }]
+    }],
+
+    constructor: function() {
```

Comment:
Out of interest why is this required? Shouldn't the default constructor be called anyway?

Reply:
That's just a left over from debugging. I've removed it now in the latest commit.